Automattic/wp-calypso
# Shortlinks: Don't Show Jetpack Sidebar if Unavailable

#### Changes proposed in this Pull Request

On Pages and CPTs -- where the Shortlink Gutenberg plugin isn't available -- don't show the Jetpack sidebar.

(Previously, the Shortlink plugin would unconditionally render `<JetpackPluginSidebar />`, even when it wasn't available. This would lead to an empty Jetpack sidebar on pages.)

#### Screenshots

##### Before

![image](https://user-images.githubusercontent.com/96308/52444382-7c8fdd00-2b28-11e9-9781-c2645925c2b2.png)

###### After

![image](https://user-images.githubusercontent.com/96308/52444566-0049c980-2b29-11e9-995e-b33a06a13859.png)

#### Testing instructions

Navigate to a page, and verify that the Jetpack sidebar isn't available.
Compare to a post, and verify that it still is there.

Fixes #30657 (reported by @danjjohnson)
